pysc2/lib/features.py

```diff
@@ -318,6 +320,25 @@ def __init__(self,
         fl_opts = game_info.options.feature_layer
         self._feature_screen_px = point.Point.build(fl_opts.resolution)
         self._feature_minimap_px = point.Point.build(fl_opts.minimap_resolution)
+        
+        # For feature units
```

Comment:
It'd be nice if this observation could be used without a game_info object, so move this to a _init_camera method and take map_size and camera_width_world_units in the constructor.